beluga_ros/src/ndt_ellipsoid.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright 2024 Ekumen, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <beluga_ros/ndt_ellipsoid.hpp>
+
+#include <Eigen/Core>
+#include <beluga/eigen_compatibility.hpp>
+
+namespace beluga_ros {
+
+namespace {
+
+template <typename RealScalar>
+inline bool isApprox(RealScalar x, RealScalar y, RealScalar prec = Eigen::NumTraits<RealScalar>::dummy_precision()) {
```

Comment:
style: Use snake_case for free functions here and everywhere else.

```suggestion
inline bool is_approx(RealScalar x, RealScalar y, RealScalar prec = Eigen::NumTraits<RealScalar>::dummy_precision()) {
```